.claude/skills/frontend-dev-guidelines/resources/data-fetching.md

```
@@ -1,33 +1,35 @@
 # Data Fetching Patterns
 
-Data fetching architecture: loaders, actions, queries, mutations, and repositories.
+Data fetching architecture in TanStack Start: route loaders, server functions, queries, mutations, and repositories.
 
 ---
 
 ## Architecture
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)